app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Api::V1::FactsController < ApplicationController
+    before_action :set_fact, only: [:show, :update, :destroy]
+
+    # GET /users/:user_id/facts
+    def index
+      @user = User.find(params[:user_id])
+      render json: @user.facts # note that because the facts route is nested inside users
+                               # we return only the facts belonging to that user
+    end
+  
+    # GET /users/:user_id/facts/:id
+    def show
+      render json: @fact, status: 200# your code goes here
+    end
+  
+   
+  
+    # POST /users/:user_id/facts
+    def create
+       @user = User.find(params[:user_id])
+      @fact = @user.facts.new(fact_params)
+      if @fact.save
+      flash.notice = "The fact record was created successfully."
```

Comment:
flash notice is not recognized by rails when there is no redirect, in this case it is unnecessary and should be removed.